modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/page_attachment_action.jsp

```diff
@@ -65,18 +65,12 @@ WikiPage wikiPage = WikiPageAttachmentsUtil.getPage(attachmentsFileEntry.getFile
 		</c:when>
 		<c:otherwise>
 			<c:if test="<%= WikiPagePermission.contains(permissionChecker, wikiPage, ActionKeys.DELETE) %>">
-				<portlet:actionURL name="/wiki/edit_page_attachment" var="deleteURL">
-					<portlet:param name="<%= Constants.CMD %>" value="<%= trashHelper.isTrashEnabled(scopeGroupId) ? Constants.MOVE_TO_TRASH : Constants.DELETE %>" />
-					<portlet:param name="redirect" value="<%= currentURL %>" />
-					<portlet:param name="nodeId" value="<%= String.valueOf(wikiPage.getNodeId()) %>" />
-					<portlet:param name="title" value="<%= wikiPage.getTitle() %>" />
-					<portlet:param name="fileName" value="<%= HtmlUtil.unescape(attachmentsFileEntry.getTitle()) %>" />
-				</portlet:actionURL>
-
-				<liferay-ui:icon-delete
-					trash="<%= trashHelper.isTrashEnabled(scopeGroupId) %>"
-					url="<%= deleteURL %>"
-				/>
+				<div class="delete-attachment" data-cmd="<%= trashHelper.isTrashEnabled(scopeGroupId) ? Constants.MOVE_TO_TRASH : Constants.DELETE %>" data-fileName="<%= attachmentsFileEntry.getTitle() %>" data-nodeId="<%= String.valueOf(wikiPage.getNodeId()) %>" data-rowId="<%= attachmentsFileEntry.getFileEntryId() %>" data-title="<%= wikiPage.getTitle() %>">
```

Comment:
Hi @IstvanD, 
don't we need here the `data-ticketKey` param?

Reply:
Hi @ambrinchaudhary 

I was thinking about it, but I could not manage to create the ticket per Wikipage, only per attachment, as ```page_attachment_action.jsp``` is rendered on each attachment.

Reply:
ok, I see... but I am just wondering, is that param really necessary? I don't see it in the previous actions url's?  What do you think @robertoDiaz ?